bundles/org.openhab.voice.whisperstt/README.md

```diff
@@ -0,0 +1,204 @@
+# Whisper Speech-to-Text
+
+Whisper STT Service uses [whisper.cpp](https://github.com/ggerganov/whisper.cpp) to perform offline speech-to-text in openHAB.
+It also uses [libfvad](https://github.com/dpirch/libfvad) for voice activity detection to isolate single command to transcribe, speeding up the execution.
+
+[Whisper.cpp](https://github.com/ggerganov/whisper.cpp) is a high-optimized lightweight c++ implementation of [whisper](https://github.com/openai/whisper) that allows to easily integrate it in different platforms and applications.
+
+Whisper enables speech recognition for multiple languages and dialects:
+
+english, chinese, german, spanish, russian, korean, french, japanese, portuguese, turkish, polish, catalan, dutch, arabic, swedish, 
+italian, indonesian, hindi, finnish, vietnamese, hebrew, ukrainian, greek, malay, czech, romanian, danish, hungarian, tamil, norwegian, 
+thai, urdu, croatian, bulgarian, lithuanian, latin, maori, malayalam, welsh, slovak, telugu, persian, latvian, bengali, serbian, azerbaijani, 
+slovenian, kannada, estonian, macedonian, breton, basque, icelandic, armenian, nepali, mongolian, bosnian, kazakh, albanian, swahili, galician, 
+marathi, punjabi, sinhala, khmer, shona, yoruba, somali, afrikaans, occitan, georgian, belarusian, tajik, sindhi, gujarati, amharic, yiddish, lao, 
+uzbek, faroese, haitian, pashto, turkmen, nynorsk, maltese, sanskrit, luxembourgish, myanmar, tibetan, tagalog, malagasy, assamese, tatar, lingala, 
+hausa, bashkir, javanese and sundanese.
+
+## Supported platforms
+
+This add-on uses some native binaries to work.
+You can find here the used [whisper.cpp Java wrapper](https://github.com/GiviMAD/whisper-jni) and [libfvad Java wrapper](https://github.com/GiviMAD/libfvad-jni).
+
+The following platforms are supported:
+
+* Windows10 x86_64
+* Debian GLIBC x86_64/arm64 (min GLIBC version 2.31 / min Debian version Focal)
+* macOS x86_64/arm64 (min version v11.0)
+
+The native binaries for those platforms are included in the distributed jar.
```

Comment:
"distributed jar" is a little too much technical for a basic user.
Proposal: ... are included in this add-on provided with openHAB distribution.

I have not compiled the binding. What is its current size ?
In case it is very big, would it make sense that you apply the same strategy that you finally did for another voice binding, that is  dynamic loading of the appropriate library at runtime if I am not wrong ?

Reply:
The add-on size is 5.4MB.

Reply:
This is big compared to standard binding.
Do you have an alternative solution ?
@kaikreuzer : what is your opinion, does it remain an acceptable size ? If I am not wrong, you implicitly propose 10 MB as a maximum ?

Reply:
Yes, I think this size is absolutely fine. And yes, anything bigger than 10MB is probably worth a closer look.